mkt/langpacks/utils.py

```diff
@@ -0,0 +1,43 @@
+ # -*- coding: utf-8 -*-
+from django import forms
+
+from tower import ugettext as _
+
+from mkt.files.utils import WebAppParser
+
+
+# FIXME: needs test, by mocking get_json_data() in various situations.
+class LanguagePackParser(WebAppParser):
+    def parse(self, upload):
+        """Parse the FileUpload passed in argument and return langpack data.
+        May raise forms.ValidationError()"""
+        # At the moment, we don't care about most of the manifest for
+        # langpacks so we simply extract the minimum we require ; in the future
+        # we might want to call the parent's parse() method and have a way to
+        # deal with extra stuff in the child.
+        data = self.get_json_data(upload)
```

Comment:
I've added the role check below, but still need to add the opposite one in `WebAppParser` and write the tests.